boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -198,6 +198,11 @@ add_library(freeldr_common ${FREELDR_BOOTLIB_COMMON_SOURCE} ${FREELDR_ARC_COMMON
 add_pch(freeldr_common include/freeldr.h FREELDR_COMMON_SOURCE)
 add_dependencies(freeldr_common bugcodes asm xdk)
 
+if(USE_CLANG_CL)
+    # We need to reduce the binary size
+    set_property(SOURCE ${FREELDR_BOOTLIB_COMMON_SOURCE} ${FREELDR_ARC_COMMON_SOURCE} APPEND_STRING PROPERTY COMPILE_FLAGS " /Os")
```

Comment:
Isn't there a better way ? This doesn't look like a long-term solution.

Reply:
Even with this we can't boot and we get the blue screen with debug info :(